packages/components/AddCommentButton/AddCommentButton.tsx

```diff
@@ -25,22 +25,29 @@ export const AddCommentButton = forwardRef<HTMLButtonElement, AddCommentButtonPr
     const localRef = useRef<HTMLButtonElement>(null);
     const mergedRefs = mergeRefs([localRef, ref]);
     const [isHovered, setIsHovered] = useState(false);
-    const isOpened = isHovered;
+    const hoverRef = useRef<ReturnType<typeof setTimeout> | null>(null);
+    const maybeHover = () => {
+      hoverRef.current = setTimeout(() => setIsHovered(true), 300);
+    };
+    const clearHover = () => {
+      hoverRef.current && clearTimeout(hoverRef.current);
+      setIsHovered(false);
+    };
```

Comment:
If we decide to move forward with this pattern– maybe we could wrap it in a `useDebouncedHover` custom hook?

Although something feels a bit off about adding delay for pop-outs like this (to me). I totally trust your judgement on the interaction though.

Reply:
Sounds good! Yeah, I don't personally like the large slide-out label and might push for this to just be a tooltip with some of the other designs I've been exploring here.